cli/cmd/service/create.go

```diff
@@ -17,10 +23,39 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
-			res1, err := client.CreateService(cmd.Context(), &adminv1.CreateServiceRequest{
+			// Parse attributes if provided
+			var attrs map[string]string
+			if attributes != "" {
+				if err := json.Unmarshal([]byte(attributes), &attrs); err != nil {
+					return err
```

Comment:
nit: people will probably mess up here, so some context might be nice:
```suggestion
					return fmt.Errorf("failed to parse --attributes as JSON: %w", err)
```